Sources/HyphenableText/HyphenableText.swift

```diff
@@ -13,6 +13,26 @@ public extension String {
   ///
   /// - see: [Wikipedia](https://en.wikipedia.org/wiki/Soft_hyphen).
   static let softHyphen = "\u{00AD}"
+    
+  /// Split the string using space as a separator and, for those substrings
+  /// whose lengths are greater than or equal to minimumWordLength,
+  /// replace them with a softHyphenated version.
+  ///
+  /// - note: This assumes that words are delineated by
+  /// space characters, which may not be correct for all locales where
+  /// CFStringIsHyphenationAvailableForLocale(_:) returns true. Consider
+  /// using CFStringTokenizer to eliminate this assumption.
```

Comment:
Very interesting, first time I hear about this `CFStringTokenizer`, but I agree it's out of scope for this PR.